python/content_apis/objects.py

```diff
@@ -0,0 +1,90 @@
+from content_apis.helpers import response_object_or_error
+
+class Document:
+    def __init__(self, conn, bucket_name, path):
+        """ Intialise Document/Object """
+        self.conn = conn
+        self.bucket_name = bucket_name
+        self.path = path
+
+    @classmethod
+    def create(cls, conn, bucket_name, path, data, obj_type):
```

Comment:
More arguments required for this function similar to bucket create. Like: `immutable`, `lock`, `version`, `template` etc

Reply:
Done